### ISSUE TITLE
After upgrading the program cannot start

### ISSUE THREAD

**Issue by azachar** [eberlitz/splashicon-generator#24]
Hello, 
I just updated to the latest version 0.2.8 and I cannot start the program.

```
node -v
v5.5.0

splashicon-generator 
env: node\r: No such file or directory
```

Do you have any idea what is wrong?

Cheers,
Andrej

**Comment by eberlitz** [comment]
Sorry, I published the file with dos line endings. Fixed in version 0.2.9.

**Comment by azachar** [comment]
Thanks, it works now!
